src/filter.js

```diff
@@ -128,6 +143,12 @@ class MetadataFilter {
 		return ['artist', 'track', 'album', 'albumArtist'];
 	}
 
+	static assertFilterFunctionIsValid(fn) {
+		if (typeof fn !== 'function') {
+			throw new Error(`Invalid filter function: ${fn}`);
```

Comment:
```suggestion
			throw new Error(`Invalid filter function: ${typeof(fn)} given`);
```

What do you think of this? Would it help with debugging?

Reply:
Yes, you're right.

Initially there was a type, instead of an actual value.